src/AuthModule/Components/VerifyAccount/VerifyAccount.tsx

```diff
@@ -1,7 +1,158 @@
-import React from 'react'
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+import logo from "../../../assets/PMS 3.svg";
 
 export default function VerifyAccount() {
+const [userEmail, setUserEmail] = useState("")
+useEffect(() => {
+  setUserEmail(localStorage.getItem("register-Email"))
+  
+}, [])
+
+  const [isLoading, setIsLoading] = useState(false);
+  const navigate = useNavigate();
+
+
+
+async function handleRegister(values:any) {
+  try {
+  const {data} = await axios.put(`${baseUrl}/Users/verify `,values)
+    console.log(data);
+    toast.success(data?.message)
+    navigate("/")  
+    localStorage.removeItem("register-Email")
+   
+    
+  } catch (error:any) {
+    console.log(error);
+    toast.error(error?.response?.data?.message||"There's a mistake." );
+  }
+  setIsLoading(false)
+
+}
+
+function handleData(data:any) {
+const  values = {
+    email:userEmail,
+    code:data?.code
+  }
+    handleRegister(values);
+  
+}
+
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit = async (data:any) => {
+    setIsLoading(true)
+    handleData(data)
+  };
+
+ 
   return (
-    <div>verifyaccount</div>
-  )
+    <>
+      <div className="auth-container vh-100  d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2 " src={logo} alt="logo" />
+        </div>
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className="form  col-xl-5 col-lg-6 col-md-7 col-sm-9 col-10 bg-info rounded-4 p-sm-5 p-3"
+        >
+         <p className="text-light mb-0 mt-3">welcome to PMS</p>
+          <h2 className="form-Name ">
+            <span>V</span>erify Account
+          </h2>
+
+          <div className="  d-flex flex-column gap-3  mt-4">
+            {/* <div className=" one-input-group">
```

Comment:
I recommend you to delete any Unnecessary comments for better readability and better performance